sdk/eventhub/event-hubs/test/sender.spec.ts

```diff
@@ -778,6 +794,7 @@ describe("EventHub Sender", function(): void {
 
     it("can be manually traced", async function(): Promise<void> {
       const tracer = new TestTracer();
+      const existingTracer = getTracer();
```

Comment:
This almost feels like it should be in a `beforeEach` and `afterEach` step, but that's not a blocker for me.

Reply:
I was thinking the same and that we should group them together (they're in separate describe()'s).

Reply:
I added a helper in to make this a bit easier to do. It turns out that we're doing this same tracer trick in multiple files as well so that snuck up on me :)